CCIT_Portal/urls.py

```diff
@@ -15,12 +15,18 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from allauth.account.views import LoginView, LogoutView
 from pages import views as pages_views
 
+# from users.views import CustomLoginView
+
+
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("contact/", pages_views.contact, name="contact-us"),
     path("complaints/", include("complaints.urls")),
     path("home/", pages_views.home, name="home"),
+    path("", LoginView.as_view(), name="account_login"),
```

Comment:
We shouldn't have the base URL as a login view directly. Keep it at `/login` and the `/home` view should be at `/`, where it redirects to `/login` if not logged in.

Reply:
ok i will fix that